swayfmt/tests/mod.rs

```diff
@@ -586,9 +584,7 @@ fn multiline_comments() {
 }"#,
         r#"contract;
 
-fn single_comment_same_line() {
-    /* a comment */
-}
+fn single_comment_same_line() { /* a comment */ }
```

Comment:
I'm curious - if this comment were a `// a comment` instead of `/* a comment */`, would it behave the same? Or does the formatter know this is collapsible because it's a "block" comment?

Reply:
Hmm good catch. So this:

```rust
fn single_comment_same_line() { // a comment
}
```

actually doesn't newline it atm. Working on a fix

Reply:
Maybe this isn't an actual issue - rustfmt has the same result. Perhaps we can merge this first and reconsider semantics again?

Reply:
In my original comment I was just checking that this:
```rust
fn single_comment_same_line() {
    // a comment
}
```
doesn't get turned into this
```rust
fn single_comment_same_line() { // a comment }
```

If not, and if we behave the same as rustfmt, then feel free to disregard :)